client/components/eventform/eventform.controller.js

```diff
@@ -168,32 +191,81 @@ angular.module('venueApp')
       $scope.selectingEvent = false;
       $scope.eventInfo = event;
     };
+          // var bounds = map.getBounds();
+          // var ne = bounds.getNorthEast();
+          // var sw = bounds.getSouthWest();
+
+    $scope.clearMapPoly = ()=>{
+
+    };
 
     $scope.createEventInfo = (form)=>{
       $scope.submitted = true;
-      if (form.$valid && $scope.files){
+      // For each place, get the icon, place name, and location.
+      var bounds = new google.maps.LatLngBounds();
+      for (var a = 0; a < $scope.allShapes.length ; a++){
+        var shape = $scope.allShapes[a].overlay;
+
+        for (var b = 0; b < shape.getPath().getLength() ; b++){
+          bounds.extend(shape.getPath().getAt(b));
+        }
+      }
+      var allcoordinates = [];
+      for (var a = 0; a < $scope.allShapes.length ; a++){
+        if (shape.getPath().getLength() < 3){
+          continue;
+        }
+        var coords = [];
+        var poly = [];
+        var shape = $scope.allShapes[a].overlay;
+
+        for (var b = 0; b < shape.getPath().getLength() ; b++){
+          coords.push([shape.getPath().getAt(b).lng(), shape.getPath().getAt(b).lat()]);
+        }
+        coords.push([shape.getPath().getAt(0).lng(), shape.getPath().getAt(0).lat()]);
+
+        poly.push(coords);
+
+        allcoordinates.push(poly);
+      }
+
+      $scope.event.map.bounds = {
+        northeast: {
+          latitude: bounds.getNorthEast().lat(),
+          longitude: bounds.getNorthEast().lng()
+        },
+        southwest: {
+          latitude: bounds.getSouthWest().lat(),
+          longitude: bounds.getSouthWest().lng()
+        }
+      }
+
+      if (form.$valid && $scope.file && allcoordinates.length){
         $scope.eventInfo = {
           title: $scope.event.title,
           description: $scope.event.description,
           times:[{
             start: $scope.event.startDate,
             end: $scope.event.endDate
           }],
-          files: $scope.files,
+          files: [$scope.file],
           location: {
-            address: $scope.event.map.markers[0].formatted_address,
-            description: $scope.event.map.markers[0].name,
+            address: $scope.event.location.address,
+            description: $scope.event.location.description,
             geo: {
               type: 'Point',
               coordinates: [
                 $scope.event.map.center.longitude,
                 $scope.event.map.center.latitude
               ]
             },
-            radius: Math.abs($scope.event.map.bounds.northeast.longitude-$scope.event.map.bounds.southwest.longitude)
+            radius: Math.abs($scope.event.map.bounds.northeast.longitude-$scope.event.map.bounds.southwest.longitude),
```

Comment:
That was not actually a change, just a comma.
I'm not sure if radius is used anywhere anymore, but I need to check with the iOS app.

It is just the one dimension currently because its was the longer of the 2 (the map is horizontal)